examples/aws-production/cluster.lokocfg

```diff
@@ -50,6 +50,8 @@ cluster "aws" {
   dns_zone         = var.dns_zone
   dns_zone_id      = var.route53_zone_id
   ssh_pubkeys      = var.ssh_public_keys
+  os_channel       = "stable"
```

Comment:
Hm, maybe we could leave them commented out, to show to the user, that those fields are optional? Example:
```suggestion
  //os_channel       = "stable"
```

And maybe add one empty line between required and optional parameters to easy distinguish between them.

Reply:
I would also suggest doing the same change to all `examples` configuration files, so we are consistent.

Reply:
@johananl what do you think?

Reply:
I can do that.

`os_channel` and `os_version` seems to be only supported on `aws`, `baremetal` and `packet` so I will only do the change there.

Reply:
Thanks!

Reply:
Updated on aws and packet.

There are no examples on `baremetal`, so no need for changes here.